apiary.apib

```diff
@@ -1272,6 +762,17 @@ Both keys can be found on your API page in your dashboard. We recommend that you
 
 ### List all logs [GET]
 
+Returns a list of the logs 
+
+**Response body**
+
+|  Name | Description | Example |
+| ---------------| ------------| --------|
+| id | the queue id | 1 |
```

Comment:
unsure what is meant by id here, assumed queue id, as agent-id and session-id are referenced elsewhere